src/screens/CurrencySelectModal.js

```diff
@@ -166,7 +162,8 @@ export default function CurrencySelectModal() {
       if (
         otherAsset &&
         newAsset?.type !== otherAsset?.type &&
-        !hasShownWarning
+        !hasShownWarning &&
+        !crosschainEnabled
```

Comment:
i feel like we need another behavior if is enabled? since we don't have it yet, would this be breaking the flow somehow if the flag is on?